react-components/src/components/RevealContainer/RevealContext.ts

```diff
@@ -1,12 +1,15 @@
-import { Cognite3DViewer } from '@cognite/reveal';
-import {createContext, useContext} from 'react';
+/*!
+ * Copyright 2023 Cognite AS
+ */
+import { type Cognite3DViewer } from '@cognite/reveal';
+import { createContext, useContext } from 'react';
 
 export const RevealContext = createContext<Cognite3DViewer | null>(null);
 
-export const useReveal = () => {
+export const useReveal = (): Cognite3DViewer => {
   const reveal = useContext(RevealContext);
-  if (!reveal) {
+  if (reveal == null) {
```

Comment:
```suggestion
  if (reveal === null) {
```

Should there be a lint rule for this?